### PR TITLE
New website design

### PR DESCRIPTION
**Pull request by amolenaar**
This PR provides the website with a completely new design.

It has fewer pages, but all current url's are redirected to the new pages (important for SEO/Google).

There are some things I want to pick up in later commits:

* Upgrade to a newer bootstrap version (4.x, now 3.3.6)
* Upgrade jQuery, or better: remove it
* background image for article style page titles (medium-like)
* dynamically populate tutorials page by tagging the tutorials
* add some screenshots at the end of the landing page (a strip of 4 images)

---

Landing page:

- Gaphor

---

Download page:

Download - Gaphor

### CODE REVIEW

File: _usps/06-standard.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+title: Standards Compliant
+icon: anchor
+---
+
+Gaphor works according to the UML 2.x and SysML standard. SysML Safety profile and C4 are planned.
```
Comment (danyeaw) [inline _usps/06-standard.md:6]: Gaphor implements the UML and SysML standards. The Safety and Reliability Profile and the C4 model for visualizing software architectures are planned.

File: _usps/05-extensible.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Extensible
+icon: plug
+---
+
+Gaphor is extensible. Plug in a code generator or export your diagrams for documentation.
```
Comment (danyeaw) [inline _usps/05-extensible.md:6]: Plug-in

File: _usps/03-beginner-pro.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+title: Beginner or Pro
+icon: project-diagram
+---
+
+Wherther you're a casual modeler (e.g. to document a project) and Model Driven Development expert, Gaphor got you covered.
```
Comment (danyeaw) [inline _usps/03-beginner-pro.md:6]: Whether you're a casual modeler documenting a project or a Model Driven Development expert, Gaphor has got you covered.

File: index.html [hunk 4]
```diff
@@ -1,154 +1,70 @@
...
+<header id="header" class="header-front-page"
+	style="background-image: url(images/splash.jpg); background-attachment: fixed">
+
+	{% include menu.html %}
+
+	<div class="bottom-header front-page">
+		<div class="container">
+			<div class="row">
+				<div class="col-sm-12">
+					<h1>UML Modeling for Everyone</h1>
```
Comment (danyeaw) [inline index.html:17]: What about just "Modeling for Everyone"

File: index.html [hunk 5]
```diff
@@ -1,154 +1,70 @@
...
+					<a href="{% link download.html %}" title="Download"
+						class="btn header-button-two">Download</a>
+				</div>
+			</div>
+		</div>
+	</div>
+</header>
+
+{% capture about %}
+Gaphor is a UML modeling application written in Python. It is designed
```
Comment (danyeaw) [inline index.html:35]: Should we also include SysML?

File: index.html [hunk 6]
```diff
@@ -1,154 +1,70 @@
...
+	<div class="bottom-header front-page">
+		<div class="container">
+			<div class="row">
+				<div class="col-sm-12">
+					<h1>UML Modeling for Everyone</h1>
+				</div>
+				<div class="col-sm-8 col-sm-offset-2">
+					<div class="section-description">
+						<p>A picture says more than a thousand words.
+						Describe and document your applications with Gaphor to enhance knowledge sharing.</p>
```
Comment (danyeaw) [inline index.html:22]: What about "your applications and system with Gaphor"

File: index.html (hunk 4 above)
Comment (amolenaar) [reply index.html:17]: Better!

File: index.html (hunk 5 above)
Comment (amolenaar) [reply index.html:35]: Good question. Technically we have the ability to cover all modeling languages. Though most of what we have and plan for is a UML derivative.

I'll add it for now, we can always change the wording later.

File: index.html (hunk 6 above)
Comment (amolenaar) [reply index.html:22]: Updated. Now I emphasised knowledge sharing, maybe we can improve on the catch phrase? :)

**Comment by amolenaar** [comment]
@danyeaw Thanks! :blush: 

I updated the wording as you suggested.

**Pull request merged**